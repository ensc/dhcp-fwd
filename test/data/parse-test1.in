user		bin
group		bin

chroot		/var/tmp
logfile		/var/tmp/dhcp-fwd.log
loglevel	1

pidfile         /var/run/dhcp-fwd.pid


if eth0		true	false	false
if eth1		false	false	true
if eth2		true	true	false

name eth0	eth0-name
name eth1	X

ip eth0		192.168.42.23
ip eth2		10.0.0.2

server		ip 192.168.31.41
server		bcast eth0

ulimit stack	100K
ulimit data	10k
ulimit core	-1
ulimit rss	1M
ulimit nproc	0
ulimit nofile	2
ulimit memlock	10m
ulimit as	0x10
ulimit locks	010
