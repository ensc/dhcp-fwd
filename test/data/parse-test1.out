uid=1
gid=1
chroot_path='/var/tmp'
logfile_name='/var/tmp/dhcp-fwd.log'
pidfile_name='/var/run/dhcp-fwd.pid'
loglevel=1
ulimits={9, data={
  stack     -> (102400, 102400)
  data      -> (10000, 10000)
  core      -> (INF, INF)
  rss       -> (1048576, 1048576)
  nproc     -> (0, 0)
  nofile    -> (2, 2)
  memlock   -> (10000000, 10000000)
  as        -> (16, 16)
  locks     -> (8, 8)}}
interfaces={len=3, data={
  'eth0', 'eth0-name', 1, 0, 0, 192.168.42.23
  'eth1', 'X', 0, 0, 1, 255.255.255.255
  'eth2', '', 1, 1, 0, 10.0.0.2}}
servers={len=2, data={
  UNICAST, 192.168.31.41
  BROADCAST, &eth0}}
